docs/articles_en/assets/snippets/npu_remote_objects_creation.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/runtime/core.hpp>
+#include <openvino/runtime/intel_npu/level_zero/level_zero.hpp>
+#include <openvino/runtime/intel_npu/properties.hpp>
+
+int main() {
+    ov::Core core;
+    auto model = core.read_model("model.xml");
+    auto input = model->get_parameters().at(0);
+
+    auto compiled_model = core.compile_model(model, "NPU");
+    auto npu_context = compiled_model.get_context().as<ov::intel_npu::level_zero::ZeroContext>();
+
+    auto in_element_type = input->get_element_type();
+    auto in_shape = input->get_shape();
+
+    {
+        //! [default_context_from_core]
+        auto npu_context = core.get_default_context("NPU").as<ov::intel_npu::level_zero::ZeroContext>();
+        // Extract raw level_zero context handle from RemoteContext
+        void* context_handle = npu_context.get();
+        //! [default_context_from_core]
+    }
+
+    {
+        //! [default_context_from_model]
+        auto npu_context = compiled_model.get_context().as<ov::intel_npu::level_zero::ZeroContext>();
+        // Extract raw level zero context handle from RemoteContext
+        void* context_handle = npu_context.get();
+        //! [default_context_from_model]
+    }
+
+    {
+        //! [wrap_d3d12_resource]
+        void* shared_buffer = nullptr;  // create the Direct3D12 resource
```

Comment:
The shared buffer address on Windows corresponds to an NT handle (see code [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu/src/backend/src/zero_remote_tensor.cpp#L47)). How is that handle created is beyond the scope of the example. To avoid giving the impression that shared_buffer is a pointer to a DX12 resource I think we should change all the dx12 references to NT handle. After all, NT handle is the Windows equivalent to Linux file descriptor for DMA buffers.

Reply:
thanks for the explanation. done!